cmd/gitops/get/cmd.go

```diff
@@ -0,0 +1,21 @@
+package get
+
+import (
+	"github.com/go-resty/resty/v2"
+	"github.com/spf13/cobra"
+	"github.com/weaveworks/weave-gitops/cmd/gitops/get/templates"
+)
+
+func GetCommand(endpoint string, client *resty.Client) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "get",
```

Comment:
@J-Thompson12 Can you confirm this will work with our current implementation of `get commits`?

Reply:
Our current `get commits` is under `app`: `gitops app <appname> get commits`